src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -448,10 +443,9 @@ public OptionData setChannelTypes(@Nonnull Collection<ChannelType> channelTypes)
      * @param  value
      *         The minimal value which can be provided for this option.
      * @throws IllegalArgumentException
-     *         If any of the following checks fail
      *         <ul>
-     *             <li>{@link OptionType type of this option} is {@link OptionType#INTEGER INTEGER} or {@link OptionType#NUMBER NUMBER}</li>
-     *             <li>{@code value} is larger than or equal to {@link OptionData#MIN_NEGATIVE_NUMBER MIN_NEGATIVE_NUMBER}</li>
+     *             <li>If {@link OptionType type of this option} is not {@link OptionType#INTEGER INTEGER} or {@link OptionType#NUMBER NUMBER}</li>
+     *             <li>If {@code value} is lower than or not equal to {@link OptionData#MIN_NEGATIVE_NUMBER MIN_NEGATIVE_NUMBER}</li>
```

Comment:
```suggestion
     *             <li>If {@code value} is lower than {@link OptionData#MIN_NEGATIVE_NUMBER MIN_NEGATIVE_NUMBER}</li>
```

Reply:
applies to many

Reply:
Also, we need to pick a consistent terminology across all these docs
Options
 1) Greater than vs Less than
 2) Higher vs Lower
 3) Larger vs Smaller

Lets not mix the two